backend/test/integration/test_calendar.py

```diff
@@ -384,6 +385,20 @@ def test_create_first_caldav_calendar(self, with_client):
         assert 'user' not in data
         assert 'password' not in data
 
+    def test_create_google_caldav_calendar(self, with_client):
+        response = with_client.post(
+            '/caldav/auth',
+            json={
+                'url': 'https://' + GOOGLE_CALDAV_DOMAINS[0],
+                'user': os.getenv('CALDAV_TEST_USER'),
+                'password': os.getenv('CALDAV_TEST_PASS'),
+            },
+            headers=auth_headers,
+        )
+        assert response.status_code == 400, response.text
+        data = response.json()
+        assert data['detail']['id'] == 'GOOGLE_CALDAV_NOT_SUPPORTED'
+
     def test_update_existing_caldav_calendar_with_password(self, with_client, with_db, make_caldav_calendar):
```

Comment:
Awesome, thanks for adding this 🥳

Reply:
Thanks for reminding me! Feel free to do that everytime 😂